src/main/java/core/basesyntax/Drawing.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawing {
+    void drawingField();
```

Comment:
```suggestion
    void draw();
```
the name of the method should begin with verb